build/scripts/build.ps1

```diff
@@ -490,11 +490,11 @@ function Ensure-ProcDump() {
 function Redirect-Temp() {
     $temp = Join-Path $binariesDir "Temp"
     Create-Directory $temp
-    Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\.editorconfig") $temp
-    Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.props") $temp
-    Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.targets") $temp
-    Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.rsp") $temp
-    Copy-Item (Join-Path $repoDir "src\Workspaces\CoreTestUtilities\Resources\NuGet.Config") $temp
+    Copy-Item (Join-Path $RepoRoot "src\Workspaces\CoreTestUtilities\Resources\.editorconfig") $temp
+    Copy-Item (Join-Path $RepoRoot "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.props") $temp
+    Copy-Item (Join-Path $RepoRoot "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.targets") $temp
+    Copy-Item (Join-Path $RepoRoot "src\Workspaces\CoreTestUtilities\Resources\Directory.Build.rsp") $temp
+    Copy-Item (Join-Path $RepoRoot "src\Workspaces\CoreTestUtilities\Resources\NuGet.Config") $temp
```

Comment:
This change causes the script to be inconsistent about whether or not it assumes PWD is `$RepoRoot` or not. Above you made a change where this is now assumed (Restore-Packages). Here though we don't assume that.

Reply:
I'm not sure what change in `Restore-Packages` you mean. The only change there was to remove explicit argument to `Restore-Project`. But yes, that function assumes the workding dir to be `$RepoRoot`, but that was there before. The only thing I changed about `$RepoRoot` was the rename from `$repoDir`. 

I'll follow up with more changes. At the end we shouldn't need `Restore-Packages` function at all.

Reply:
Ignore this. I misread the change in Restore-Packages